doc/source/indexing.rst

```diff
@@ -500,6 +500,62 @@ A list of indexers where any element is out of bounds will raise an
 
 .. _indexing.basics.partial_setting:
 
+Selecting Random Samples
+------------------------
+.. versionadded::0.16.1
+
+A random selection of rows from a Series, DataFrame, or Panel with the ``.sample()`` method. 
+The method accepts a specific number of rows to return, or a fraction of rows. 
+
+.. ipython :: python
+
+   s = Series([0,1,2,3,4,5])
+
+   # When no arguments are passed, returns 5 rows like .head() or .tail()
+   s.sample()
+   
+   # One may specify either a number of rows:
+   s.sample(n = 3)
```

Comment:
You fixed the spacing around `=` in your code but not the documentation examples :)